docs/README.rst

```diff
@@ -108,6 +114,10 @@ General
 
    :code:`python-basedir` optionally specifies the path to an alternate |kctlr| to F5 CCCL agent (:code:`bigipconfigdriver.py`). `F5 Controller Agent`_ is the default agent.
 
+   Use the ``vs-snat-pool-name`` if you want virtual servers to reference a preconfigured SNAT pool by name in the Common partition on the BIG-IP device.
```

Comment:
So the user must create the snat pool in the Common partition, or can they create them in the controller's partition?

Reply:
They can create them in controller's partition since CCCL does not track SNAT pools, but we decided not to document this since we already recommend all preconfigured objects like health monitors be created in Common